### PR TITLE
Vbc task: avoid moving .pdb in design-time build

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
@agocke, @jasonmalinowski, @ManishJayaswal

File: src/Compilers/Core/MSBuildTask/Vbc.cs [hunk 1]
```diff
@@ -294,6 +294,19 @@ internal void MovePdbFileIfNecessary(string outputAssembly)
                 return;
             }
 
+            // Design-time builds should not have side effects.
```
Comment (jasonmalinowski) [inline src/Compilers/Core/MSBuildTask/Vbc.cs]: It seems like this is fallout based on how the Execute method is being overwritten. Perhaps a better override can be provided where something only gets called if the build actually, well, built?

Comment (cston) [inline src/Compilers/Core/MSBuildTask/Vbc.cs]: There doesn't appear to be a Task method that gets called when build completes successfully. Ideally, this issue should be fixed by adding a /pdb:file option to the VB compiler (see #3859) and removing the handling of .pdbs here. If we make a fix here in the meantime, the fix should be relatively simple.

**Comment by agocke** [comment]
:+1: If we need a spot-fix. Otherwise, I would prefer #3859 be fixed and we remove this option from the Vbc task.

**Comment by cston** [comment]
@Pilchie FYI

**Pull request closed without merging**